.travis.yml

```diff
@@ -21,7 +20,12 @@ matrix:
   include:
     - php: hhvm-3.12
       env: REMOVE_XDEBUG="0" HHVM="1"
-      dist: trusty
+    - php: 5.3
```

Comment:
You shouldn't specify 5.3 here, you can leave it up and add the `dist` key under the `php` version entry.

Reply:
Could you show me how?

```yml
language: php
sudo: false

php:
  - 5.3
    dist: precise
  - 5.4
  - 5.5
  - 5.6
  - 7.0
  - 7.1
  - nightly
```

does not work: https://travis-ci.org/nokitakaze/sentry-php/builds/285444487

Reply:
Don't you think you are mistaken? https://docs.travis-ci.com/user/reference/trusty#PHP-images official guide:
```yml
matrix:
  include:
    - php: 5.2
      dist: precise
    - php: 5.3
      dist: precise
```

Reply:
My bad, I thought otherwise.